Source/WebCore/platform/mediastream/RealtimeMediaSourceCenter.cpp

```diff
@@ -123,11 +123,18 @@ void RealtimeMediaSourceCenter::createMediaStream(Ref<const Logger>&& logger, Ne
 
 void RealtimeMediaSourceCenter::getMediaStreamDevices(CompletionHandler<void(Vector<CaptureDevice>&&)>&& completion)
 {
-    enumerateDevices(true, true, true, [this, completion = WTFMove(completion)]() mutable {
+    bool shouldEnumerateDisplay = false;
+#if USE(GLIB)
+    shouldEnumerateDisplay = true;
+#endif
```

Comment:
Rather than hardcoding support with `#if USE(GLIB)`, can you add a virtual method to `DisplayCaptureManager` and use it here an in `UserMediaPermissionRequestManagerProxy`?

Reply:
That's a good suggestion, I'll give it a try, thanks!